services/kafka-operator/0.20.0/kafka-operator.yaml

```diff
@@ -11,6 +11,7 @@ spec:
       sourceRef:
         kind: HelmRepository
         name: kubernetes-charts.banzaicloud.com
+        namespace: ${workspaceNamespace}
```

Comment:
This is used when the kommander cli extracts the helm repos, should be a noop i think